clients/client-go/http.go

```diff
@@ -259,15 +259,21 @@ func (client *Client) APICall(payload interface{}, method, route string, result
 	return result, callSummary, nil
 }
 
-// SignedURL creates a signed URL using the given Client, where route is the
-// url path relative to the RootURL stored in the Client, query is the set of
-// query string parameters, if any, and duration is the amount of time that the
-// signed URL should remain valid for.
+// SignedURL creates a signed URL using the given Client, where route is
+// either a url path relative to `<RootURL>/api/<serviceName>/<apiVersion>` or
+// a fully qualified URL.  query is the set of query string parameters, if any,
+// and duration is the amount of time that the signed URL should remain valid
+// for.  The full-URL form permits signing URLs that are not even on the given
+// RootURL.
```

Comment:
It would be good to mention in the last sentence why signing foreign URLs is useful/desirable.

Reply:
I'm not sure it is useful / desirable -- I just added this to avoid dropping the feature from tc-proxy.